internal/service/location.go

```diff
@@ -0,0 +1,23 @@
+package service
+
+import (
+	"go-rest-template/internal/domain"
+	"go-rest-template/internal/repository"
+)
+
+type LocationService interface {
+	Create(location domain.Payload) domain.Payload
```

Comment:
Recibe el puntero, devuelve un puntero.
```suggestion
	Create(p *domain.Payload) *domain.Payload
```